### ISSUE TITLE
BigDecimal('0.0') / BigDecimal('0.0') returns nan instead of NaN in IRB

### ISSUE THREAD

**Issue by deepj** [oracle/truffleruby#1683]
### 😹 

To reproduce:

Run irb and then

```ruby
require 'bigdecimal'
BigDecimal('0.0') / BigDecimal('0.0')
```

TruffleRuby 1.0.0 RC16

```
=> nan
```

MRI 2.6.3

```
=> NaN
```

**Comment by nirvdrum** [comment]
Fixing this should be easy, but is it actually breaking anything?

**Comment by deepj** [comment]
Nope, but it's be nice to be fixed

**Comment by nirvdrum** [comment]
This was easy enough to fix. The commit with the fix is 0299e6ae3f9b76f793b245042dbd0cd663ac945f along with new specs to ensure it doesn't regress. Unfortunately I didn't pick up this issue until the next release was already in code freeze, so you won't see it until the release after that one.
